program/idl.ts

```diff
@@ -26,6 +26,9 @@ import {
 } from "codama";
 import { writeFileSync } from "fs";
 import { SYSTEM_PROGRAM_ADDRESS } from "@solana-program/system";
+import { MPL_TOKEN_METADATA_PROGRAM_ID } from "@metaplex-foundation/mpl-token-metadata";
+import { TOKEN_2022_PROGRAM_ID } from "@solana/spl-token";
```

Comment:
I don't think these two constants warrant two additional dependencies that depend on the enormous legacy `@solana/web3.js`.

I'm guessing this is why you had to mess with the `pnpm-workspace.yaml` file.

I don't think we have a generated Kit client for Token Metadata just yet (I believe @mcintyre94 is working on that) so I'd just inline the constant since this is never gonna change anyway.

I'd personally do the same for the Token 2022 program but if you wanted to import it, consider using `@solana-program/token-2022` instead which is Kit compatible and does not import `@solana/web3.js`.